cli/src/main/java/io/github/bmarwell/jdtfmt/format/FormatProcessor.java

```diff
@@ -16,10 +17,13 @@
 
 public class FormatProcessor {
 
+    private final OutputWriter writer;
     private final FormatterMode fmtMode;
     private final DiffOptions diffOptions;
 
-    public FormatProcessor(FormatterMode fmtMode, DiffOptions diffOptions) {
+    public FormatProcessor(OutputWriter
+    public FormatProcessor(Writer writer, FormatterMode fmtMode, DiffOptions diffOptions) {
```

Comment:
This part of the diff looks very odd (at least in GitHub) - I even doubt it compiles. No workflow has attempted to build this branch, which might've hidden it.

Reply:
Oh, what are the odds... :-D